src/Eto.WinForms/CustomControls/TreeController.cs

```diff
@@ -485,7 +377,7 @@ public void ExpandToItem (ITreeGridItem value)
 			var parents = GetParents (value).Reverse ();
 
 			foreach (var item in parents) {
-				var row = IndexOf (item);
+				var row = treeDataStore.IndexOf(item);
 				if (row >= 0 && !IsExpanded(row))
```

Comment:
Instead of `IsExpanded()` why not use `item.Expanded`?

```suggestion
				if (row >= 0 && !item.Expanded)
```